### PR TITLE
[CDS-80416]: fix static schema for steps in custom stage

### PR DESCRIPTION
**Pull request by shalini-agr**

### CODE REVIEW

**Review by mohitg0795 — APPROVED** [review]

File: v1/pipeline/steps/custom/custom-approval-step-info.yaml [hunk 1]
```diff
@@ -3,42 +3,42 @@ allOf:
...
-  - retryInterval
-  - scriptTimeout
+  - approval_criteria
+  - interval
+  - timeout
   - shell
   - source
   properties:
-    approvalCriteria:
+    approval_criteria:
```
Comment (archit-harness) [inline v1/pipeline/steps/custom/custom-approval-step-info.yaml:12]: shall we rename to criteria?

File: v1/pipeline/steps/custom/output-alias.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+properties:
+  key:
+    type: string
+    minLength: 1
+  scope:
+    type: string
+    enum:
+      - pipeline
+      - stage
+      - step-group
```
Comment (archit-harness) [inline v1/pipeline/steps/custom/output-alias.yaml:15]: i think its group now

Comment (shalini-agr) [reply v1/pipeline/steps/custom/output-alias.yaml]: done

File: v1/pipeline/steps/custom/harness-approval-step-info.yaml [hunk 3]
```diff
@@ -4,39 +4,20 @@ allOf:
...
       minLength: 1
-    approverInputs:
+    inputs:
       type: array
       items:
         $ref: approver-input-info.yaml
     approvers:
       $ref: approvers.yaml
-    includePipelineExecutionHistory:
+    include_pipeline_execution_history:
```
Comment (archit-harness) [inline v1/pipeline/steps/custom/harness-approval-step-info.yaml:18]: should we rename this to include_execution_history ?

File: v1/pipeline/steps/custom/harness-approval-step-info.yaml [hunk 4]
```diff
@@ -4,39 +4,20 @@ allOf:
 - type: object
   required:
   - approvers
-  - includePipelineExecutionHistory
+  - include_pipeline_execution_history
```
Comment (archit-harness) [inline v1/pipeline/steps/custom/harness-approval-step-info.yaml:7]: same as above

File: v1/pipeline/steps/custom/harness-approval-step-info.yaml (hunk 3 above)
Comment (shalini-agr) [reply v1/pipeline/steps/custom/harness-approval-step-info.yaml]: done

File: v1/pipeline/steps/custom/harness-approval-step-info.yaml (hunk 4 above)
Comment (shalini-agr) [reply v1/pipeline/steps/custom/harness-approval-step-info.yaml]: done

**Review by archit-harness — APPROVED** [review]

**Pull request merged**